build-aux/generate.mk

```diff
@@ -35,6 +35,7 @@ generate/files      += $(OSS_HOME)/pkg/envoy-control-plane/  # recipe in _cxx/en
 # Individual files: Misc
 generate/files      += $(OSS_HOME)/DEPENDENCIES.md
 generate/files      += $(OSS_HOME)/DEPENDENCY_LICENSES.md
+generate-fast/files += .github/dependabot.yml
```

Comment:
does this mean it is only re-generated when running `make generate-fast` or will it also be when we run `make generate`?

Reply:
I believe both

Reply:
`make generate` does everything that `make generate-fast` does.